package.json

```diff
@@ -33,7 +33,7 @@
 		"publish-to-npm": "npm install && npm test && npm publish --access=public"
 	},
 	"dependencies": {
-		"@natlibfi/melinda-record-import-commons": "^1.3.0",
+		"@natlibfi/melinda-record-import-commons": "^3.1.1",
```

Comment:
Update to the newest version (API authentication has changed and the new version of API client handles this).